Code/GraphMol/Atom.h

```diff
@@ -163,65 +164,38 @@ class RDKIT_GRAPHMOL_EXPORT Atom : public RDProps {
   }
   //! returns the explicit degree of the Atom (number of bonded
   //!   neighbors in the graph)
-  /*!
-    <b>Notes:</b>
-      - requires an owning molecule
-  */
   unsigned int getDegree() const;
 
   //! returns the total degree of the Atom (number of bonded
   //!   neighbors + number of Hs)
-  /*!
-    <b>Notes:</b>
-      - requires an owning molecule
-  */
   unsigned int getTotalDegree() const;
 
   //! \brief returns the total number of Hs (implicit and explicit) that
   //! this Atom is bound to
-  /*!
-    <b>Notes:</b>
-      - requires an owning molecule
-  */
   unsigned int getTotalNumHs(bool includeNeighbors = false) const;
 
   //! \brief returns the total valence (implicit and explicit)
   //! for an atom
-  /*!
-    <b>Notes:</b>
-      - requires an owning molecule
-  */
   unsigned int getTotalValence() const;
 
   //! returns the number of implicit Hs this Atom is bound to
-  /*!
-    <b>Notes:</b>
-      - requires an owning molecule
-  */
   unsigned int getNumImplicitHs() const;
 
+  //! returns the valence (explicit or implicit) of this atom
```

Comment:
Even though it is obvious, I would mark the parameter here since it will show up in code completion
```
// \param getExplicit if true, return the explicit valence
```